policyengine_us/parameters/gov/states/ct/tax/income/credits/agi/head_of_household.yaml

```
@@ -0,0 +1,134 @@
+description: Connecticut provides the following credit amount under the AGI credit, based state AGI for head of household filers.
```

I'm guessing here, but in general the description should express what the values are and not abbreviate
```suggestion
description: Connecticut provides head of household filers a credit for this fraction of taxes, depending on their adjusted gross income.
```